meson_options.txt

```
@@ -1,6 +1,7 @@
 option('man-pages', type: 'feature', value: 'auto', description: 'Generate and install man pages')
 option('xwayland', type: 'feature', value: 'auto', description: 'Enable support for X11 applications')
 option('svg', type: 'feature', value: 'enabled', description: 'Enable svg window buttons')
+option('xpm', type: 'feature', value: 'disabled', description: 'Enable xpm window buttons')
```

I think "window icons" is better as we won't support `{close,max,...}.xpm`.

```suggestion
option('xpm', type: 'feature', value: 'disabled', description: 'Enable xpm window icons')
```